staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go

```diff
@@ -73,11 +73,23 @@ func (obj *Unstructured) UnstructuredContent() map[string]interface{} {
 	}
 	return obj.Object
 }
+
+// UnstructuredContent returns a map contain an overlay of the Items field onto
+// the Object field. Items always overwrites overlay. Changing "items" in the
+// returned object will not affect the map.
```

Comment:
the comment is misleading... changing "items" in the returned object does affect `obj.Object`, right? and the objects in the returned `items` list are also not deep copies